server/source/configDB/dbConnect.js

```diff
@@ -0,0 +1,8 @@
+import pg from 'pg';
+
+const databaseURL = 'postgresql://postgres:key07202020@localhost:5432/barefoot';
```

Comment:
I think this should be declared in `.env` file because others may have different credentials to access the DB.

Reply:
Yeah Definitely, I wish to have that `.env` file to keep those data but you know this task was done by Christian. So hope after merge Christian will move this configuration into his `.env` file. if am not mistaken

Reply:
Why pushing this URL. It has to stay in the .env file, you just need it locally right.

Reply:
You know we didn't have .env file because this task of to configure .env file was done by other Teammate. But I hope everyone after pull from develop he/she will add those configurations into .env file.